usaspending_api/database_scripts/matview_generator/mv_contract_award_search.json

```diff
@@ -9,6 +9,12 @@
     "  awards.category,",
     "  awards.type,",
     "  awards.type_description,",
+    "  awards.generated_unique_award_id,",
+    "  CASE",
+    "    WHEN awards.piid IS NOT NULL THEN awards.piid  -- contracts. Did it this way to easily handle IDV contracts",
```

Comment:
`piid` won't be null for contracts or IDVs The case statement can be removed

Reply:
`display_award_id` is now always `awards.piid` in the `mv_idv_award_search`

Reply:
`display_award_id` is now always `awards.piid` in `mv_idv_award_search`

Reply:
It looks the same:

```sql
    CASE
    WHEN awards.piid IS NOT NULL THEN awards.piid  -- contracts. Did it this way to easily handle IDV contracts
    ELSE NULL
    END AS display_award_id,
```

It should be reduced to `awards.piid AS display_award_id` for both of the procurement materialized views

Reply:
Made this change for `mv_contract_award_search` as well